ONSdigital/census-rm-sample-loader
# Download File From Sample Bucket

# Motivation and Context
Slicker process to get file from sample loader bucket to folder

# What has changed
New script to access file in bucket and download to specific folder in pod

# How to test?
Build docker image
Run/Terraform from Ryan's branches on other card:
https://github.com/ONSdigital/census-rm-kubernetes/compare/1520-sort-out-the-sample-loader-deployment
https://github.com/ONSdigital/census-rm-terraform/compare/1520-sort-out-the-sample-loader-deployment
Put a file in the new bucket that has been created
Run new script in sample-loader in new pod (bash) and check that the sample file from the bucket is downloaded into specific folder

# Links
https://trello.com/c/vrKXfWLF/1521-allow-the-sample-loader-to-download-file-from-the-sample-bucket-5
https://trello.com/c/lKt8leWZ/1520-sort-out-the-sample-loader-deployment-make-it-deployable-with-a-manifest-file-5
